lib/cli.js

```diff
@@ -28,7 +28,26 @@ exports._parse = function (argv, _console, callback) {
       'Delay between validation requests to avoid service blacklisting, defaults to ' + DEFAULT_SLEEP_MS + 'ms',
       DEFAULT_SLEEP_MS)
     .option('--concurrency <concurrency>',
-      'Amount of requests to run in parallel, defaults to ' + DEFAULT_CONCURRENCY, DEFAULT_CONCURRENCY);
+      'Amount of requests to run in parallel, defaults to ' + DEFAULT_CONCURRENCY, DEFAULT_CONCURRENCY)
+    .option('--usermedium <usermedium>',
+      'Medium where the CSS will be used, defaults to `all`. Possible values are `aural`, `braille`, ' +
```

Comment:
We should probably explicitly list the default variable in this `.option`. The rationale is that if the site changes its default, then we're improperly stating the default in this text

Reply:
Deciding to take even lower maintenance route of removing the listing and telling people to look at the website for default and options. It's silly for us to list a default that might change and isn't in our control

Reply:
Going to include a few options for demonstration/sanity of user